eclair-core/src/main/scala/fr/acinq/eclair/io/Switchboard.scala

```diff
@@ -72,10 +72,8 @@ class Switchboard(nodeParams: NodeParams, watcher: ActorRef, relayer: ActorRef,
     case authenticated: PeerConnection.Authenticated =>
       // if this is an incoming connection, we might not yet have created the peer
       val peer = createOrGetPeer(authenticated.remoteNodeId, offlineChannels = Set.empty)
-      val features = nodeParams.overrideFeatures.get(authenticated.remoteNodeId) match {
-        case Some(f) => f
-        case None => nodeParams.features.maskFeaturesForEclairMobile()
-      }
+      // TODO: @tbast: we now apply maskFeaturesForEclairMobile to custom features as well, is that ok?
```

Comment:
I think this is ok. The two features we mask off are really invoice features; the only place where it really matters to set them is inside invoices.
We can omit them in `Init`, I'm not seeing any issue it could cause.

Reply:
```suggestion
```